src/main/java/org/commcare/suite/model/RemoteQueryDatum.java

```diff
@@ -46,10 +47,12 @@ public RemoteQueryDatum() {
     public RemoteQueryDatum(URL url, String storageInstance,
             List<QueryData> hiddenQueryValues,
                             OrderedHashtable<String, QueryPrompt> userQueryPrompts,
-                            boolean useCaseTemplate, boolean defaultSearch, boolean dynamicSearch, Text title, Text description) {
+                            boolean useCaseTemplate, boolean defaultSearch, boolean dynamicSearch, Text title, Text description,
```

Comment:
nit: formatting seems a bit off here.

Reply:
Is this what you meant? [2b4ba65](https://github.com/dimagi/commcare-core/pull/1389/commits/2b4ba65b1ffdeaea1429db098d7b44f3d66391b2)